### ISSUE TITLE
can i the change the FrozenBatchNorm2d by torch.nn.BatchNorm2d？

### ISSUE THREAD

**Issue by dreamhighchina** [tianzhi0549/FCOS#182]
i want to know why FrozenBatchNorm2d frozens the batch to 1？could i change it by torch.nn.BatchNorm2d？？？thanks a lot...........

**Comment by tianzhi0549** [comment]
@dreamhighchina `FrozenBatchNorm2d` has nothing to do with the batch size. Compared to `nn.BatchNorm2d,` `ForzenBatchNorm2d` uses pre-trained running mean and variance and does not update them.

**Comment by dreamhighchina** [comment]
thank you for your reply。
